File: Java/08-02/t06.java
package t_02_08;

import java.util.Scanner;

public class t06 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter the principle amount  :  ");
		double p = sc.nextDouble();
		System.out.println("Enter the interest rate in % : ");
		double r = sc.nextDouble();
		System.out.println("Enter the time period in years : ");
		double t = sc.nextDouble();
	
		System.out.println("The simple interest is " + (double)(p*r*t/100));
		}
}

File: Kotlin/02-09/task/t06.kt
fun main() {
    print("Enter the principle amount : ")
    var p = readLine()!!.toDouble()
    print("Enter the rate amount in %: ")
    var r = readLine()!!.toDouble()
    print("Enter the time period in years: ")
    var t = readLine()!!.toDouble()
    println("The simple interest is " + (p*r*t/100))
}
